node/pkg/watchers/evm/ccq.go

```diff
@@ -19,6 +20,30 @@ import (
 	"github.com/certusone/wormhole/node/pkg/query"
 )
 
+// ccqStart starts up CCQ query processing using the configured number of worker routines.
+func (w *Watcher) ccqStart(ctx context.Context, errC chan error) {
+	if w.ccqTimestampCache != nil && w.ccqBackfillCache {
+		w.ccqBackfillStart(ctx, errC)
+	}
+
+	for count := 0; count < w.ccqConfig.NumWorkers; count++ {
+		workerId := count
+		common.RunWithScissors(ctx, errC, fmt.Sprintf("evm_fetch_query_req_%d", workerId), func(ctx context.Context) error {
+			w.ccqLogger.Debug("CONCURRENT: starting worker", zap.Int("worker", workerId))
+			for {
+				select {
+				case <-ctx.Done():
+					return nil
+				case queryRequest := <-w.queryReqC:
+					w.ccqLogger.Debug("CONCURRENT: processing query request", zap.Int("worker", workerId))
```

Comment:
Since the workerIds will repeat for all the chains, does the logger identify which watcher this is?  I wasn't sure about the ccqLogger.

Reply:
I think when setting up the logger it does 
[```w.ccqLogger = logger.With(zap.String("component", "ccqsol"))```](https://github.com/wormhole-foundation/wormhole/blob/0248837bf374a762b57560699edde83ef92f1248/node/pkg/watchers/solana/ccq.go#L35)
[```w.ccqLogger = logger.With(zap.String("component", "ccqevm"))```](https://github.com/wormhole-foundation/wormhole/blob/0248837bf374a762b57560699edde83ef92f1248/node/pkg/watchers/evm/watcher.go#L203-L204)

Reply:
Yeah, all of the watcher loggers have the chain name in them. However, I have reconsidered having the worker number in the runnable name. It really adds no value and just pollutes the scissors metrics. Please see the rework commit and comment again if you have any other concerns.